README.md

```diff
@@ -46,6 +46,9 @@ You need to:
 - cleanup FATS, typically:
   - `source ./cleanup.sh`
 
+### Running locally
+
+To skip installing the tools that have been installed locally set the environment variable SKIP_INSTALLED to true
```

Comment:
Is this sufficient to run locally? Seems like people might be in for a rude awakening if the scripts do something they don't expect, like dropping resource from their cluster.

It may also make sense to split the run and cleanup scripts to be more like how system splits starting the cluster/registry, installing components into the cluster and running tests.

It probably makes the most sense for users to run the test functions locally, but not the install components.